tests/test_extensions.py

```diff
@@ -0,0 +1,132 @@
+"""
+Unit tests for Extension Manager
+"""
+
+import sys
+import os
+import unittest
+import mock
+import shutil
+
+dir_of_executable = os.path.dirname(__file__)
```

Comment:
Not sure why would a user want to run the tests. The only people who should run the test are the developers. The developers should run tox on each change to make sure nothing is broken prior to making a PR.